ext/bcmath/libbcmath/src/debug.c

```diff
@@ -32,11 +32,10 @@
 #include <stdio.h>
 #include "bcmath.h"
 #include "private.h"
-#include <stddef.h>
```

Comment:
This header is used for ``size_t``

Reply:
I reverted those headers. They were covered indirectly with `bcmath.h` though.